src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The function 'calculateRentalCost' uses hard-coded values for specific numbers of days, which is not scalable or maintainable. Use constants for the daily cost and discounts, and implement logic to calculate the total cost based on the number of days rented.